phy/cluster/manual/session.py

```diff
@@ -201,18 +188,36 @@ def _store_cluster(self,
                               append=True,
                               )
 
-    def store_all_clusters(self, spikes_per_cluster):
+    def is_consistent(self, cluster, spikes):
+        """Return whether the data of a cluster is consistent."""
```

Comment:
I don't disagree. But it's still misleading to imply it's a data consistency check! I reckon MD5 hashing each cluster would probably be more hassle / time than it's worth.